arangod/Aql/DocumentProducingNode.cpp

```diff
@@ -39,14 +40,14 @@
 using namespace arangodb::aql;
 
 namespace {
-std::string_view const filterKey("filter");
-std::string_view const maxProjectionsKey("maxProjections");
-std::string_view const producesResultKey("producesResult");
+constexpr std::string_view filterKey("filter");
+constexpr std::string_view producesResultKey("producesResult");
 }  // namespace
 
 DocumentProducingNode::DocumentProducingNode(Variable const* outVariable)
     : _outVariable(outVariable),
       _count(false),
+      _useCache(true),
```

Comment:
This flag only has an effect if there exists a cache for an index.
If no cache exists, then the value of this flag does not matter. However, if a cache exists, it will be used by default, except this flag here is set to `false`.